presto-main/src/test/java/com/facebook/presto/type/TestRealOperators.java

```diff
@@ -186,6 +187,16 @@ public void testCastToVarchar()
     }
 
     @Test
+    public void testCastToChar()
+            throws Exception
+    {
+        assertFunction("CAST(REAL'754.1985' as CHAR)", CHAR, "754.1985");
```

Comment:
`CHAR` has length of 1. I would expect this to fail (I would expect length is 8)